src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxUI.java

```diff
@@ -454,6 +454,31 @@ protected LayoutManager createLayoutManager() {
     }
 
     class AquaComboBoxLayoutManager extends BasicComboBoxUI.ComboBoxLayoutManager {
+        protected Rectangle rectangleForCurrentValue() {
+            int width = comboBox.getWidth();
+            int height = 22;
+            Insets insets = getInsets();
+            int buttonSize = height - (insets.top + insets.bottom);
+            if ( arrowButton != null )  {
+                buttonSize = arrowButton.getWidth();
+            }
+            int midHeight = (comboBox.getHeight() - height - (insets.top + insets.bottom)) / 2 - 1;
+            if (midHeight < 0) {
+                midHeight = 0;
+            }
+
+            if (comboBox.getComponentOrientation().isLeftToRight()) {
+                return new Rectangle(insets.left, insets.top + midHeight,
```

Comment:
OK. Since Apple has confirmed combobox height shouldn't change with font size, we can have it fixed.

Only thing I guess, till we sort out JDK-8023912: [macosx] JComboBox RTL orientation problem
lets use LTR as it seems it might give a false impression that we support RTL when we dont even know if Apple supports it or not. Do we see any difference with LTR and RTL?

Reply:
I don't see a difference either but was following the same logic as used in BasicComboBoxUI. I can remove the LTR vs RTL logic until the other issue is resolved if needed.

Reply:
You can keep this for now...You can take in JDK-8023912 incase which I guess needs a followup question to Apple.